modules/database/base/base.py

```
@@ -0,0 +1,192 @@
+
+    @false_if_error
+    def _create_entry(self, table_schema, **kwargs) -> object:
+        entry = table_schema(**kwargs)
+        self.session.add(entry)
+        self._commit()
+        return entry
+
+    @false_if_error
+    def _create_entry_if_does_not_exist(self, table_schema, fields_to_check: List[str], **kwargs) -> object:
+        filters = {k: v for k, v in kwargs.items() if k in fields_to_check}
+        entries = self._get_all_entries(table_schema=table_schema, *filters)
```

Ruff (0.8.0)

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)